.azure-pipelines/azure-pipelines.yml

```diff
@@ -149,8 +147,6 @@ stages:
               test: fedora37
             - name: openSUSE 15
               test: opensuse15
-            - name: Ubuntu 20.04
-              test: ubuntu2004
```

Comment:
These lines should not be removed. We only want to remove the Ubuntu 20.04 VM, not containers.